docs/translation.md

```diff
@@ -53,9 +52,8 @@ If you have any questions or need assistance, feel free to reach out to the lang
 
 * German: [Robin Meese](https://github.com/robson90)
 * Dutch: [Jon Koops](https://github.com/jonkoops)
-* French: [Dimitri](https://github.com/DimitriDR) && [Gilbert Fernandes](https://github.com/gilbert-fernandes)
-* Catalan: [jmallach](https://github.com/jmallach) && [Ecron](https://github.com/Ecron)
 * Japanese: [y-tabata](https://github.com/y-tabata) && [wadahiro](https://github.com/wadahiro)
+* Japanese: [herver1971](https://github.com/herver1971) && [anthieni](https://github.com/anthieni)
```

Comment:
Pretty sure this should not be Japanese, likely a copy paste error?

Reply:
@jonkoops yeah it is. Good that catched it.

Fixed.